docs/usage/exporting_models.md

```diff
@@ -84,6 +84,17 @@ as an optional alternative which implements ISO 8601 time diff encoding.
 See [below](#custom-json-deserialisation) for details on how to use other libraries for more performant JSON encoding
 and decoding.
 
+## `model.construct()`
+
+Using the construct function, it is possible to instantiate a model without validation.
```

Comment:
```suggestion
Using the `construct` function, it is possible to instantiate a model without validation.
```
We need a more complete description of why you might use `construct` and the risks of using it, but if you're not sure what to say, leave it and I'll do this later.